### PR TITLE
Add Helm chart for Proxy

### LINKED ISSUE #21: Add helm chart to proxy

### PR DESCRIPTION
**Pull request by zpatrick**
closes #21

### CODE REVIEW

File: proxy/helm/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Helm Chart for Orca Proxy
+
+## Install
+To install this helm chart into your cluster, navigate to this directory and run `helm install`:
+```console 
+$ cd orca/proxy/helm
+$ helm install --name orca-proxy orca-proxy
```
Comment (nehayward) [inline proxy/helm/README.md]: Add note about `helm tiller start` if not installed.

File: proxy/helm/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+If installing into a Minikube cluster, you can run the `minikube service list` to discover the endpoint for the Orca Proxy:
+```console
+$ minikube service list
+|-------------|----------------------|---------------------------|
+|  NAMESPACE  |         NAME         |            URL            |
+|-------------|----------------------|---------------------------|
+| default     | kubernetes           | No node port              |
+| kube-system | kube-dns             | No node port              |
+| kube-system | kubernetes-dashboard | http://192.168.64.5:30000 |
+| kube-system | orca-proxy           | http://192.168.64.5:32404 |
```
Comment (nehayward) [inline proxy/helm/README.md]: Do we need a not about export/set this env variable to `REACT_APP_K8S_API_CLIENT_BASE_URL`

Comment (zpatrick) [reply proxy/helm/README.md]: the proxy doesn't have any configuration values like that, I'm assuming that's for the frontend?

Comment (nehayward) [reply proxy/helm/README.md]: I believe I'm just confused on if this replaces having to run `kubectl proxy -p 8080` or if its another component

File: proxy/helm/README.md (hunk 1 above)
Comment (zpatrick) [reply proxy/helm/README.md]: Kind of including your comment the other day about adding a command to setup minikube here:

We generally leave out documentation on getting dependencies setup out of our codebases unless there is a really strong case for it. I know it's only one line in this instance, but we've churned over this in the past and determined that the best rule of thumb is something like: 
* Give the reader a solid list of dependencies/requirements (and links to those tools if necessary). 
* Assume the reader knows how to get those setup in their dev environment better than we do. 

We usually just list dependencies at the top of the README, but I feel that's redundant given we are already in the `minikube` section of the `helm` README.  


That being said, I do think we ought to have a **Dependencies** section in the top-level README at some point, but figured we would tackle that once we do a serious pass-through of that README.

Comment (nehayward) [reply proxy/helm/README.md]: That's a solid reason, the only issue I have is when you give a user a command prompt to copy and paste and then it doesn't work. Maybe a middle ground would be something like 

adding `*` to specify that it requires decencies or letting the user know this isn't a strict copy and paste for setup

File: proxy/helm/README.md (hunk 2 above)
Comment (zpatrick) [reply proxy/helm/README.md]: yep it replaces it

Comment (nehayward) [reply proxy/helm/README.md]: Okay, and it's currently not functional because auth is not in rosturm on this PR correct.

Comment (zpatrick) [reply proxy/helm/README.md]: Right - the proxy is fully functional, but we still need to wire up your auth0 changes w/ the frontend.

Comment (nehayward) [reply proxy/helm/README.md]: Okay sweet, I was just held up on testing.

File: proxy/helm/README.md (hunk 1 above)
Comment (zpatrick) [reply proxy/helm/README.md]: 👍 added a link in the top of this doc

**Pull request merged**